devicon.json

```diff
@@ -2484,6 +2484,21 @@
         "color": "#F18803",
         "aliases": []
     },
+    {
+        "name": "perl",
+        "tags": [
+            "programming",
+            "language"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "plain"
+            ]
```

Comment:
You'll need to add this attribute in the `versions` as well:
```
"font": [
  "plain"
]
```

Reply:
I would recommend checking the neighbouring entry to ensure all the attributes are correct. You can also check the [`CONTRIBUTING.md`](https://github.com/devicons/devicon/blob/master/CONTRIBUTING.md#updateDevicon) for more details